### DISCUSSION TITLE
Help please

### DISCUSSION THREAD

**Discussion by Xestance** [community/community#19407] [Other Feature Feedback, Questions, & Ideas]
I am a neurosurgeon and encountered a source code for a software called tactics by atami. https://github.com/Atamai/tactics
I have managed to download CMAKE, VTK 9, ITK and QT. 

I have tried a different codes using terminal to install the code and run it. I can not finish it always an error and I just need help to how to run it.

P.S. I am running Mac Catalina and I have no previous experts in softwares, coding or anything I just can follow steps.

Yours,

**Comment by doguedogue** [comment]
The best way is creating a new issue doc, I'm going to try installing all this stuff the app requiered  🚀

**Comment by Zerotask** [comment]
As a general advice you should always post steps to reproduce and error context (message, stack etc).

**Comment by rilysh** [comment]
> I am a neurosurgeon and encountered a source code for a software called tactics by atami. https://github.com/Atamai/tactics I have managed to download CMAKE, VTK 9, ITK and QT.
> 
> I have tried a different codes using terminal to install the code and run it. I can not finish it always an error and I just need help to how to run it.
> 
> P.S. I am running Mac Catalina and I have no previous experts in softwares, coding or anything I just can follow steps.
> Yours,

It will be really helpful if you provide the stack trace.

**Comment by Xestance** [comment]
CMake Deprecation Warning at CMakeLists.txt:7 (cmake_minimum_required):
  Compatibility with CMake < 2.8.12 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.


Found Boost: /usr/local/lib/cmake/Boost-1.78.0/BoostConfig.cmake (found suitable exact version "1.78.0")  
Found Boost: /usr/local/lib/cmake/Boost-1.78.0/BoostConfig.cmake (found suitable exact version "1.78.0") found components: serialization 
CMake Deprecation Warning at /usr/local/lib/cmake/vtk-9.1/vtk-use-file-deprecated.cmake:1 (message):
  The `VTK_USE_FILE` is no longer used starting with 8.90.
Call Stack (most recent call first):
  CMakeLists.txt:89 (include)


Found Boost: /usr/local/lib/cmake/Boost-1.78.0/BoostConfig.cmake (found suitable exact version "1.78.0")  
Found Boost: /usr/local/lib/cmake/Boost-1.78.0/BoostConfig.cmake (found suitable exact version "1.78.0") found components: serialization 
CMake Deprecation Warning at /usr/local/lib/cmake/vtk-9.1/vtk-use-file-deprecated.cmake:1 (message):
  The `VTK_USE_FILE` is no longer used starting with 8.90.
Call Stack (most recent call first):
  /usr/local/lib/gdcm-3.0/UseGDCM.cmake:26 (include)
  CMakeLists.txt:95 (include)


CMake Deprecation Warning at thirdparty/airs/CMakeLists.txt:4 (CMAKE_MINIMUM_REQUIRED):
  Compatibility with CMake < 2.8.12 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value or use a ...<max> suffix to tell
  CMake that the project does not need compatibility with older versions.


Found Boost: /usr/local/lib/cmake/Boost-1.78.0/BoostConfig.cmake (found suitable exact version "1.78.0")  
Found Boost: /usr/local/lib/cmake/Boost-1.78.0/BoostConfig.cmake (found suitable exact version "1.78.0") found components: serialization 
CMake Deprecation Warning at /usr/local/lib/cmake/vtk-9.1/vtk-use-file-deprecated.cmake:1 (message):
  The `VTK_USE_FILE` is no longer used starting with 8.90.
Call Stack (most recent call first):
  thirdparty/airs/CMakeLists.txt:16 (INCLUDE)


CMake Error at thirdparty/airs/CMakeLists.txt:96 (INCLUDE):
  INCLUDE could not find requested file:

    /vtkWrapPython.cmake


CMake Error at thirdparty/airs/CMake/AIRSCommonBlock.cmake:67 (VTK_WRAP_PYTHON3):
  Unknown CMake command "VTK_WRAP_PYTHON3".
Call Stack (most recent call first):
  thirdparty/airs/ImageRegistration/CMakeLists.txt:48 (INCLUDE)


Configuring incomplete, errors occurred!
See also "/Users/elkazaz/Documents/GitHub/tactics/build/CMakeFiles/CMakeOutput.log".
